swabbie-aws/src/main/kotlin/com/netflix/spinnaker/swabbie/aws/autoscalinggroups/AmazonAutoScalingGroupHandler.kt

```diff
@@ -152,20 +151,7 @@ class AmazonAutoScalingGroupHandler(
     if (serverGroups == null || serverGroups.isEmpty()) {
       return
     }
-    val elapsedTimeMillis = measureTimeMillis {
```

Comment:
No need to do this here since. We have all references already present. Load balancer and instances